docs/src/main/asciidoc/dev-ui-v2.adoc

```diff
@@ -0,0 +1,244 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Dev UI
```

Comment:
That's typically a guide which should follow the diatixys template - I would say reference guide.

Reply:
I copied from the old dev ui documentation. What is dialixys and where can I lean about how this should look ?